### PR TITLE
Update workshop module descriptions

### PR DESCRIPTION
**Pull request by loganharbour**
Refs #21192

Just in time.

### CODE REVIEW

**Review by GiudGiud — DISMISSED** [review]

File: tutorials/darcy_thermo_mech/doc/content/workshop/modules/modules.md [hunk 1]
```diff
@@ -4,24 +4,31 @@
...
+!media contact/ironing_gallery.mp4
+
+!---
+
 ## External PETSc Solver
 
-The External PETSc Solver module provides support for stand-alone native PETSc applications that
-to be coupled with moose-based applications.
+Provides support for stand-alone native PETSc applications that
+are I alto be coupled with moose-based applications. Is used as a general
```
Comment (GiudGiud) [inline tutorials/darcy_thermo_mech/doc/content/workshop/modules/modules.md:30]:
```suggestion
are I alto be coupled with moose-based applications. Is used as a general
```
what does I alto mean

Comment (loganharbour) [reply tutorials/darcy_thermo_mech/doc/content/workshop/modules/modules.md:30]: magic

**Review by GiudGiud — DISMISSED** [review]

File: tutorials/darcy_thermo_mech/doc/content/workshop/modules/modules.md [hunk 2]
```diff
@@ -4,24 +4,46 @@
...
+  style=margin:auto;
+  caption=Frictional ironing model using mortar contact.
+
+!---
+
+## Electromagnetics (coming soon)
+
+The Electromagnetics module provides components and models to simulate electromagnetic wave problems
+using MOOSE, and facilitate coupling of electromagnetic simulations to other physical domains.
+Maxwell's Equations is solved for complex fields using a Helmholtz wave equation formulation in 1-D
```
Comment (GiudGiud) [inline tutorials/darcy_thermo_mech/doc/content/workshop/modules/modules.md:33]:
```suggestion
Maxwell's equations are solved for complex fields using a Helmholtz wave equation formulation in 1-D
```

File: tutorials/darcy_thermo_mech/doc/content/workshop/modules/modules.md [hunk 3]
```diff
@@ -4,24 +4,46 @@
...
+  caption=Electric field radiation pattern of half-wave dipole antenna.
+
+!---
+
 ## External PETSc Solver
 
-The External PETSc Solver module provides support for stand-alone native PETSc applications that
-to be coupled with moose-based applications.
+Provides support for stand-alone native PETSc applications that
+are to be coupled with moose-based applications. Is used as a general
```
Comment (GiudGiud) [inline tutorials/darcy_thermo_mech/doc/content/workshop/modules/modules.md:45]:
```suggestion
are to be coupled with moose-based applications. It is also a general
```

File: tutorials/darcy_thermo_mech/doc/content/workshop/modules/modules.md [hunk 4]
```diff
@@ -46,13 +76,30 @@ separate app [!citep](Wendt2018a,Wendt2017c,Kerby2017).
...
+!--
+
 ## Heat Conduction
 
 Basic utilities for solving the transient heat conduction equation:
 
 !equation
 \rho c_p \pf{T}{t} - \nabla\cdot k \nabla T - s = 0
 
+Also contains capability for generalized heat transfer. Will likely be renamed
```
Comment (GiudGiud) [inline tutorials/darcy_thermo_mech/doc/content/workshop/modules/modules.md:100]:
```suggestion
Also contains capability for generalized heat transfer (convection, radiation, ...). Will likely be renamed
```

**Review by GiudGiud — APPROVED** [review]

**Pull request merged**